doc/admin-guide/files/records.config.en.rst

```diff
@@ -1084,6 +1084,21 @@ mptcp
    This enables buffering the content for incoming ``POST`` requests. If enabled no outbound
    connection is made until the entire ``POST`` request has been buffered.
 
+.. ts:cv:: CONFIG proxy.config.http.request_line_max_size INT 65535
+
+   Controls the maximum size, in bytes, of an HTTP Request Line in requests. Requests
+   with a request line which exceed this size will cause the entire request to be
```

Comment:
exceeds

Reply:
Hmm..the text starts with _Requests.._ anyway, reworded the sentence.